experimental/traffic-portal/server.ts

```diff
@@ -34,6 +34,14 @@ import {
 
 import { AppServerModule } from "./src/main.server";
 
+const versionParts = process.versions.node.split(".");
+if (versionParts.length !== 3) {
+	console.warn(`Unable to determine current node version, got ${versionParts.length} items`);
+} else if (versionParts[0] < "16" || (versionParts[0] === "16" && versionParts[1] < "20")) {
+	console.error(`Unsupported version of node found: ${process.version}, expected >=16.20`);
+	process.exit(1);
+}
+
```

Comment:
We shouldn't have to check version information in the code like this, and it creates more overhead for version upgrades. Is this part needed, and if so, why?

Reply:
It's not _needed_, but more of an anti-foot gun measure. I'd argue for keeping it in, but most of the issues this would likely "catch" aren't actually catch-able as the file probably won't parse in that node version (so the code will never get ran).